MailCore/Models/Draft.swift

```diff
@@ -336,13 +344,28 @@ public extension Draft {
         guard let signatureNode = try? document.getElementsByClass(Constants.signatureWrapperIdentifier).first() else {
             return !document.hasText()
         }
+
+        // We check if the signature was changed. If it was edited, we do want to save the draft.
+        // User could have taped inside the signature div and started a Dostoevsky class novel.
+        let signatureNodeText: String? = try? signatureNode.text()
+        guard let rawSignature,
```

Comment:
Instead of storing the raw signature in Realm, maybe we can compare the current signature (`signatureNodeText`) and the signature stored in realm (thanks to `identityId`)?

Reply:
Yeah but then what should we do if the signature is removed from the server (and also the phone), or just changed in mid air when we do perform the check ?